src/nginx/conf/nginx-config.template

```diff
@@ -522,6 +534,11 @@ stream {
     access_log /var/lib/nginx/log/stream_access.log stream;
           {{ end }}
     proxy_pass be_stream_{{$cfg.Namespace}}_{{$cfg.Name}};
+    {{ if isTrue (env "ROUTER_ENABLE_UNSAFE_ANNOTATIONS") }}
```

Comment:
Could you add an empty line before `{{ if isTrue (env "ROUTER_ENABLE_UNSAFE_ANNOTATIONS") }}` to improve readability?

Reply:
Adjusted in 9cbb472